downstream/modules/platform/proc-configure-controller-OCP.adoc

```diff
@@ -0,0 +1,106 @@
+[id="configure-controller-OCP"]
+
+During a Kubernetes upgrade, {ControllerName} has to be running. The following procedure outlines the configuration changes you need to make in automation controller to minimize downtime during the upgrade.
+
+.Prerequisites
+
+* {PlatformNameShort} 2.4
+* Ansible {ControllerName} 4.4
+* {OCPShort}
+** 4.10.42
+** 4.11.16
+** 4.12.0
+* High availability (HA) deployment of Postgres
+* Multiple worker node that {ControllerName} pods can be scheduled on
+
+.Procedure
+
+. Enable `RECEPTOR_KUBE_SUPPORT_RECONNECT` in {ControllerName} spec.
```

Comment:
`{ControllerName}`

This is specifically refering to the kubernetes resource type AutomationController